be/src/olap/tablet.cpp

```diff
@@ -2047,6 +2052,26 @@ void Tablet::_rowset_ids_difference(const RowsetIdUnorderedSet& cur,
     }
 }
 
+// The caller should hold _rowset_update_lock and _meta_lock lock.
+Status Tablet::update_delete_bitmap_without_lock(const RowsetSharedPtr& rowset) {
+    int64_t cur_version = rowset->start_version();
+    std::vector<segment_v2::SegmentSharedPtr> segments;
+    _load_rowset_segments(rowset, &segments);
+
+    DeleteBitmapPtr delete_bitmap = std::make_shared<DeleteBitmap>(tablet_id());
+    RETURN_IF_ERROR(calc_delete_bitmap(rowset->rowset_id(), segments, nullptr, delete_bitmap,
+                                       cur_version - 1, true));
+
+    for (auto iter = delete_bitmap->delete_bitmap.begin();
```

Comment:
Can we use the DeleteBitmap::merge method directly? As well as in another update_delete_bitmap method?